src/components/SelectInput/SelectInput.tsx

```diff
@@ -1,20 +1,20 @@
-import React, { FC, FocusEvent } from 'react';
+import React, { FC, FocusEvent, ReactNode } from 'react';
 import classNames from 'classnames';
 import Select, { ValueType, OptionTypeBase, FocusEventHandler } from 'react-select';
 import FormLabel from '../FormLabel/FormLabel';
 import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
 import styles from './SelectInput.module.scss';
 
-interface SimulatedEventPayload {
+type SimulatedEventPayloadType = {
```

Comment:
I believe that this is just a type, that happens to be used in an interface, as well as elsewhere.